ood-test.osc.edu/apps/myjobs/templates/Basic_CUDA_Serial_Job_Pitzer/manifest.yml

```diff
@@ -0,0 +1,6 @@
+name: Basic CUDA Serial Job - Pitzer
+host: pitzer
+script: pbs_script.sh
```

Comment:
CUDA still stays "Queued" for a long time for me, and only for Pitzer, not Owens. But this might be an issue that we can't resolve here.

Reply:
Yeah, it does take some time in the queue and I'm not sure how to improve. After the wait, it will be successfully completed.